content/id/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,702 @@
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: block-pv
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block
+  persistentVolumeReclaimPolicy: Retain
+  fc:
+    targetWWNs: ["50060e801049cfd1"]
+    lun: 0
+    readOnly: false
+```
+### _Persistent Volume Claim_ meminta Volume _Raw Block_ 
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: block-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block
+  resources:
+    requests:
+      storage: 10Gi
+```
+### Spesifikasi _Pod_ yang menambahkan alamat Perangkat _Raw Block_ pada kontainer
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-with-block-volume
+spec:
+  containers:
+    - name: fc-container
+      image: fedora:26
+      command: ["/bin/sh", "-c"]
+      args: [ "tail -f /dev/null" ]
+      volumeDevices:
+        - name: data
+          devicePath: /dev/xvda
+  volumes:
+    - name: data
+      persistentVolumeClaim:
+        claimName: block-pvc
+```
+
+{{< note >}}
+Ketika menambahkan sebuah perangkat _raw block_ untuk sebuah _Pod_, kita menspesifikasi alamat perangkat dalam kontainer alih-alih alamat pemasangan.
+{{< /note >}}
+
+### Mengikat _Block Volume_
+
+Jika seorang pengguna meminta sebuah volume _raw block_ dengan mengindikasikannya menggunakan kolom `volumeMode` pada _spec_ `PersistentVolumeClaim`, aturan pengikatannya sedikit berbeda dibanding rilis-rilis sebelumnya yang tidak memerhatikan mode ini sebagai bagian dari _spec_.
+Di bawah merupakan tabel dari kemungkinan kombinasi yang pengguna dan admin dapat spesifikasikan untuk meminta sebuah perangkat _raw block_. Tabel tersebut mengindikasikan apakah volume akan terikat atau tidak jika dikombinasikan dengan cara tertentu:
+Matriks pengikatan volume untuk volume yang disediakan secara statis:
+
+| PV volumeMode | PVC volumeMode  | Hasil            |
+| --------------|:---------------:| ----------------:|
+|   unspecified | unspecified     | TERIKAT          |
+|   unspecified | Block           | TIDAK TERIKAT    |
+|   unspecified | Filesystem      | TERIKAT          |
+|   Block       | unspecified     | TIDAK TERIKAT    |
+|   Block       | Block           | TERIKAT          |
+|   Block       | Filesystem      | TIDAK TERIKAT    |
+|   Filesystem  | Filesystem      | TERIKAT          |
+|   Filesystem  | Block           | TIDAK TERIKAT    |
+|   Filesystem  | unspecified     | TERIKAT          |
+
+{{< note >}}
+Hanya volume yang disediakan secara statis yang didukung untuk rilis alfa. Administrator harus memperhatikan nilai-nilai tersebut ketika mengerjakan perangkat-perangkat _raw block_.
+{{< /note >}}
+
+## _Volume Snapshot_ dan Dukungan Pemulihan Volume dari _Snapshot_
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Fitur _volume snapshot_ ditambahkan hanya untuk mendukung CSI Volume Plugins. Untuk lebih detail, lihat [_volume snapshots_](/docs/concepts/storage/volume-snapshots/).
```

Comment:
make CSI Volume Plugins italicized

Reply:
Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe